mmv1/products/orgpolicy/api.yaml

```diff
@@ -0,0 +1,99 @@
+    base_url: '{{name}}'
+    update_verb: :PATCH
+    min_version: beta
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/resource-manager/docs/organization-policy/creating-managing-custom-constraints'
+        'Supported Services':
+          'https://cloud.google.com/resource-manager/docs/organization-policy/custom-constraint-supported-services'
+      api: 'https://cloud.google.com/resource-manager/docs/reference/orgpolicy/rest/v2/organizations.constraints'
+    description: |
+      Custom constraints are created by administrators to provide more granular and customizable control over the specific fields that are restricted by your organization policies.
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: parent
+        description: |
+          The parent of the resource.
+        input: true
+        url_param_only: true
+        required: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        required: true
+        input: true
+        description: |
+          Immutable. The name of the custom constraint. This is unique within the organization. Format of the name should be `organizations/{organization_id}/customConstraints/{custom_constraint_id}`.
+      - !ruby/object:Api::Type::String
+        name: 'displayName'
+        description: |
+          A human-friendly name for the constraint.
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: |
+          A human-friendly description of the constraint to display as an error message when the policy is violated.
+      - !ruby/object:Api::Type::String
+        name: 'condition'
+        required: true
+        description: |
+          A CEL condition that refers to a supported service resource, for example `resource.management.autoUpgrade == false`. For details about CEL usage, see [Common Expression Language](https://cloud.google.com/resource-manager/docs/organization-policy/creating-managing-custom-constraints#common_expression_language).
+      - !ruby/object:Api::Type::Enum
+        name: 'actionType'
+        required: true
+        description: |
+          The action to take if the condition is met.
+        values:
+          - :ALLOW
+          - :DENY
+      - !ruby/object:Api::Type::Array
+        name: 'methodTypes'
+        min_size: 1
+        required: true
+        description: |
+          A list of RESTful methods for which to enforce the constraint. Can be `CREATE`, `UPDATE`, or both. Not all Google Cloud services support both methods. To see supported methods for each service, find the service in [Supported services](https://cloud.google.com/resource-manager/docs/organization-policy/custom-constraint-supported-services).
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::Array
+        name: 'resourceTypes'
+        input: true
+        min_size: 1
```

Comment:
I think we can technically add `max_size: 1` here, but in case it changes in the future, I decided to leave that validation to the server here.

Reply:
👍